src/pages/404.tsx

```diff
@@ -0,0 +1,55 @@
+import { COLOR } from '@src/styles/color';
+import { FONT_STYLES } from '@src/styles/fontStyle';
+import Link from 'next/link';
+import styled from 'styled-components';
+
+function Custom404() {
+  return (
+    <StCustom404>
+      <StBackground>
+        <StText>
+          찾을 수 없는 페이지입니다.
+          <br />
+          입력한 주소가 맞는지 확인하신 후, 딜리버블과 다시 함께 해요!
+        </StText>
```

Comment:
```suggestion
        <StText>
          <p>찾을 수 없는 페이지입니다.</p>
          <p>입력한 주소가 맞는지 확인하신 후, 딜리버블과 다시 함께 해요!</p>
        </StText>
```

저는 텍스트가 여러개일 때 이렇게 쓰고, 문장 간 간격을 gap 속성을 통해서 조절해주었어요!

Reply:
p 태그는 하나의 문단을 묶어주는 태그이고 즉, 문단을 나누기 위한 목적으로 단지 태그가 끝나는 부분에서 줄이 바뀌는 기능을 포함하고 있을 뿐 단순히 줄을 바꾸기 위해서 사용하는 태그가 아니다.
br 태그는 줄 바꿈을 목적으로 하는 태그이다. 

어떻게 작성하는 게 현재 코드에 더 적절한지 고민이 되는데 각 줄을 문단이라고 보아도 무방한 것 같아서 제안해주신 코드를 사용하는 것도 괜찮은 것 같습니다!